spec/tokenized-buffer-spec.coffee

```diff
@@ -580,7 +572,7 @@ describe "TokenizedBuffer", ->
 
     describe "when the selector matches a run of multiple tokens at the position", ->
       it "returns the range covered by all contigous tokens (within a single line)", ->
-        expect(tokenizedBuffer.bufferRangeForScopeAtPosition('.function', [1, 18])).toEqual [[1, 6], [1, 28]]
+        expect(tokenizedBuffer.bufferRangeForScopeAtPosition('.meta.function', [1, 18])).toEqual [[1, 6], [1, 28]]
```

Comment:
@ypresto You're right. Sorry about that. I switched to using `ScopeSelector` from `first-mate` without recalling that their matching criteria is stricter. Can you switch to `.string.regexp` until I can get a fix in today?